services/clients/firebase_client.py

```diff
@@ -51,7 +52,7 @@ def update_documentation(self, doc_id: str, data) -> None:
 
     @staticmethod
     def get_blob_url(blob_name, folder="repo") -> str:
-        return f"/{folder}/{blob_name}"
+        return f"{folder}/{blob_name}"
```

Comment:
I think you were following what I set up in Firebase Storage, which (for some reason) was a folder named `/` which inside of it has another folder `repo` which we store our data in. Meaning the absolute path of our data was `//repo/{id}`.

By removing this `/` it turns into just `/repo/{id}`. No clue how I made that `/` folder or why.